packages/synapse-react-client/src/synapse-client/SynapseClient.ts

```diff
@@ -5313,7 +5313,7 @@ export function getPortalFileHandleServletUrl(
  */
 export const getFeatureFlags = () => {
   return doGet<FeatureFlags>(
-    `Portal/featureflags`,
+    `/Portal/featureflags`,
```

Comment:
in the accounts app in development mode, pointing to production, we were making calls to `https://synapse.orgPortal/featureflags`, which threw an error. It seems like this actually works in real production/staging, so not sure why I was seeing it, but I would rather have 1-2 slashes than 0-1. It will be obvious if the MFA_REQUIRED feature flag breaks in staging, so this can be easily reverted if it causes problems.